apps/src/templates/courseOverview/TeacherCourseOverview.tsx

```diff
@@ -144,18 +144,12 @@ const TeacherCourseOverview: React.FC = () => {
       );
       return;
     }
-
+    let courseVersionName = selectedSection.courseVersionName;
     if (selectedSection.courseVersionName !== params.courseVersionName) {
-      navigate(
-        generatePath('../' + TEACHER_NAVIGATION_PATHS.courseOverview, {
-          courseVersionName: selectedSection.courseVersionName,
-        }),
-        {replace: true}
-      );
-      return;
+      courseVersionName = params.courseVersionName;
     }
 
-    courseSummaryCachedLoader(selectedSection.courseVersionName)
+    courseSummaryCachedLoader(courseVersionName)
```

Comment:
Can we just use `courseSummaryCachedLoader(params.courseVersionName)` instead of this new logic and variable?